run/make_init_polynya.py

```diff
@@ -0,0 +1,50 @@
+from make_init_base import initMaker
+
+# Creates initial conditions for the Bjornsson et al. (2001) polynya case
+
+# Domain size [km]
+x = 100
+y = 50
+res = 2
+
+
+nfirst = int(y / res)
+nsecond = int(x / res)
```

Comment:
How about integer division specifically, to avoid the type conversion?
```suggestion
nfirst = y // res/
nsecond = x // res
```

Reply:
Done